src/v/kafka/server/group.cc

```diff
@@ -3218,4 +3222,102 @@ std::ostream& operator<<(std::ostream& o, const group::offset_metadata& md) {
     return o;
 }
 
+bool group::subscribed(const model::topic& topic) const {
+    if (_subscriptions.has_value()) {
+        return _subscriptions.value().contains(topic);
+    }
+    // answer conservatively
+    return true;
```

Comment:
rational for this? If we don't have the topic in our subscriptions list and `_subscriptions` is nullopt wouldn't it make sense to return false?

Reply:
Seen this was asked and answered above, resolving.